config.properties

```diff
@@ -68,7 +68,8 @@ IBM_TTS_PASS: ""
 CONTEXT_GUI_ENABLED: false
 ACTION_CLIENT_SCRIPT: "/home/roboy/workspace/Roboy/src/roboy_dialog/resources/scripts/action_client.py"
 
-TELEGRAM_API_TOKENS_FILE: "resources/telegram.json"
+TELEGRAM_API_TOKENS_FILE:   "resources/telegram.json"
+EXTERNAL_API_KEY_FILE:      "resources/knowledgebase/api.key"
```

Comment:
can you pls specify what kind of API is this

Reply:
The idea was to store all sorts of keys here, like a general key storage file for all current and future services. The idea was that `APIhandler`'s `keyGetter` internal class automatically grabs your key for you from the file. 

Would `EXTERNAL_API_KEY_LIST_FILE` be better? Or would a comment be better?